README.md

```diff
@@ -162,11 +164,10 @@ The following instructions will help you go from metrics to logs to traces.
 The following instructions will help you go from metrics to logs to traces.
 
 1. In Grafana, go to the Explore view.
-2. From the data source drop-down select "Prometheus-Exemplars".
-3. Run the following query.
+2. Run the following query.
```

Comment:
Might be worth still keeping a step to make sure the right source is selected?

Reply:
Good point. Mimir _should_ be the default now, but can't hurt to be sure.